test/NeoSharp.VM.Test/VMJsonTestBase.cs

```diff
@@ -0,0 +1,108 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using NeoSharp.VM.TestHelper;
+using NeoSharp.VM.TestHelper.Extensions;
+
+namespace NeoSharp.VM.Test
+{
+    public abstract class VMJsonTestBase
+    {
+        public void ExecuteTest(string json)
+        {
+            var ut = json.JsonToVMUT();
+
+            foreach (var test in ut.Tests)
```

Comment:
I am questioning is it `Tests` or `Steps` of a single test?

Reply:
Steps is better

Reply:
This is tests, actions is steps :P